src/services/message.js

```diff
@@ -264,21 +264,18 @@ class MessageService {
     // If this is an admin then process any admin commands
     // if adminDMonly is false
 
-    if (isAdmin) {
-      if (this.adminDMonly && this.convoType !== 'personal') {
+    if (isAdmin && this.adminDMonly && message.includes('/admin')) {
```

Comment:
This code is wrong.  You are assuming that all admin commands are /admin, this is NOT true.  There are /verify commands as well, which ONLY admins have.

Also, most integrations do NOT use/set the adminDMonly value.

Reply:
admin dm only is being set as false by default in the api.